x/btcstaking/types/multi_staking_allow_list_testnet.go

```diff
@@ -0,0 +1,16 @@
+//go:build testnet
+
+package types
+
+// TODO TBD
+const multiStakingAllowListExpirationHeight = 0
+
```

Comment:
maybe we can have separate package like `types/allowlist` for all those hardcoded files ? The introduce a lot of noise in `types` package

Reply:
done